### ISSUE TITLE
PERMITTIVITY_REL and PERMEABILITY_EM_REL are quantities, not units

### ISSUE THREAD

**Issue by Toni100** [qudt/qudt-public-repo#389]
We currently have:
http://qudt.org/vocab/unit/PERMITTIVITY_REL
http://qudt.org/vocab/unit/PERMEABILITY_EM_REL

Those are not units, but quantities.
There seem to exist corresponding quantities already:
http://qudt.org/vocab/quantitykind/RelativePermittivity
http://qudt.org/vocab/quantitykind/ElectromagneticPermeabilityRatio

Corresponding Wikidata items with detailed references to IEC 80000-6:
https://www.wikidata.org/wiki/Q4027242
https://www.wikidata.org/wiki/Q77785645

**Comment by steveraysteveray** [comment]
You are right. Not only that, but the dimensionality of the unit entry for relative permittivity is wrong anyway, since it is dimensionless. Big mess!
I'll remove the units. If you feel like helping, maybe you could check the dimension vector listed for the unit entry and see if that is correct for permittivity. If so, I could add it for the numerator dimension vector and denominator dimension vector for  http://qudt.org/vocab/quantitykind/RelativePermittivity. (Same pattern as for http://qudt.org/vocab/quantitykind/ElectromagneticPermeabilityRatio)

**Comment by Toni100** [comment]
The correct dimensions for (electromagnetic) permittivity and permeability are given here:
https://www.wikidata.org/wiki/Q211569#P4020
https://www.wikidata.org/wiki/Q28352#P4020
Those are machine-verified, which is enabled by the presence of the Wolfram Language quantity ID [P7431].

(One day I might write a cross-checker which includes QUDT dimensions.)
